subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AbstractSmokeTest.groovy

```diff
@@ -93,10 +93,10 @@ abstract class AbstractSmokeTest extends Specification {
         static tomcat = "2.7.0"
 
         // https://plugins.gradle.org/plugin/io.spring.dependency-management
-        static springDependencyManagement = "1.0.11.RELEASE"
+        static springDependencyManagement = "1.0.13.RELEASE"
 
         // https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-gradle-plugin
-        static springBoot = "2.5.5"
+        static springBoot = "2.7.3"
```

Comment:
Any reason we're jumping to `2.7.x` instead of using `2.6.11` which presumably has the same change?

Reply:
We needed to jump to 2.7.2 for other changes, and then needed additional changes made in 2.7.2, so we have to use last week's 2.7.3.

Reply:
It doesn't really make a difference to me, IDC. I was just curious since it seems 2.6.11 was released on the same day and since it is seemingly actively supported, I thought they have also backported the changes in 2.7.3 to 2.6.x

Reply:
I didn't realize they were maintaining a 2.6 line as well.  2.7 is still needed here though, Sterling and I have talked about updating this a few times now.